core/ui/src/main/kotlin/org/mifospay/core/ui/TransactionItemScreen.kt

```diff
@@ -93,20 +75,35 @@ fun TransactionItemScreen(
             TransactionType.CREDIT -> "+ $formattedAmount"
             else -> formattedAmount
         }
-        Text(
-            modifier = Modifier.weight(.3f),
-            text = amount,
-            style = TextStyle(
-                fontSize = 16.sp,
-                fontWeight = FontWeight.Normal,
-                color = when (transaction.transactionType) {
-                    TransactionType.DEBIT -> red
-                    TransactionType.CREDIT -> green
-                    else -> Color.Black
-                },
-                textAlign = TextAlign.End,
-            ),
-        )
+        Row {
+            Image(
+                modifier = Modifier
+                    .size(20.dp)
+                    .padding(top = 2.dp),
+                painter = painterResource(
+                    id = when (transaction.transactionType) {
+                        TransactionType.DEBIT -> R.drawable.core_ui_money_out
+                        TransactionType.CREDIT -> R.drawable.core_ui_money_in
+                        else -> R.drawable.core_ui_money_in
+                    },
+                ),
+                contentDescription = null,
+                colorFilter = ColorFilter.tint(MaterialTheme.colorScheme.onSurface),
+            )
+            Text(
+                text = amount,
+                style = TextStyle(
+                    fontSize = 16.sp,
+                    fontWeight = FontWeight.Normal,
+                    color = when (transaction.transactionType) {
+                        TransactionType.DEBIT -> red
+                        TransactionType.CREDIT -> green
+                        else -> Color.Black
+                    },
+                    textAlign = TextAlign.End,
+                ),
+            )
+        }
     }
```

Comment:
The changes made to this file can be removed, as the design was already implemented in the previous PR (Home UI).

Reply:
Ok i will remove it